ui/Comments/CommentBlock.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { createRef } from "react";
+import { View, FlatList, TextInput, StyleSheet } from "react-native";
+import { Overlay } from "react-native-elements";
+
+import MaximisedComment from "./MaximisedComment";
+import MainComment from "./MainComment";
+import ReplyBox from "./ReplyBox";
+import BackButton from "./BackButton";
+
+type Props = {
+  onBack: () => void;
+  comment: IComment;
+};
+
+const CommentBox = (props: Props) => {
+  let inpRef: React.RefObject<TextInput> = createRef();
```

Comment:
You probably want to use a `useRef` here, instead of `createRef`. Read this [SO answer](https://stackoverflow.com/a/54620836/6199068).